judges/github-events/github-events.rb

```diff
@@ -133,6 +133,14 @@ def self.fill_up_event(fact, json)
       case json[:payload][:action]
       when 'created'
         skip_event(json) if json[:payload][:pull_request][:user][:id].to_i == fact.who
+        if Fbe.fb.query(
+          "(and (eq repository #{fact.repository}) " \
+          '(eq what "pull-was-reviewed") ' \
+          "(eq who #{fact.who}) " \
+          "(eq issue #{json[:payload][:pull_request][:number]}))"
+        ).each.last
```

Comment:
@Yegorov this looks like a pretty dirty hack :) Maybe we can better analyze the incoming JSON (from GitHub) and understand the difference between two events? They send us them both, for some reason. We just need to understand the reason.

Reply:
@yegor256 not sure, differences only in `id`. Look [this diff](https://www.diffchecker.com/1DcBl2f5/)